### DISCUSSION TITLE
Next TSC Meeting - Thu 2024-08-29 @ 1300 UTC

### DISCUSSION THREAD

**Discussion by planetf1** [pq-code-package/tsc#93] [Announcements]
Our next meeting follows on the usual two-week schedule.

**Comment by planetf1** [comment]
I've published the proposed agenda. please review and add any suggested topics.

Specifically for this week I've proposed reviewing where we are with the generic implementation & thinking about future algorithm additions such as ML-DSA

**Comment by franziskuskiefer** [comment]
I may not be able to make it today.
